src/xdist/remote.py

```diff
@@ -86,6 +136,9 @@ def sendevent(self, name, **kwargs):
         self.log("sending", name, kwargs)
         self.channel.send((name, kwargs))
 
+    def send_log(self, record):
+        self.sendevent("runtest_logmessage", record=record)
```

Comment:
Small suggestion to make things easier to understand:

```suggestion
    def send_log(self, pickled_record):
        self.sendevent("runtest_logmessage", pickled_record=pickled_record)
```